languages/csharp/exercises/concept/expression-bodied-members/.docs/after.md

```diff
@@ -0,0 +1,70 @@
+Many [types of struct and class members][expression-bodied-members] (fields being the primary exception) can use the expression-bodied member syntax. Defining a member with an expression often produces more concise and readable code than traditional blocks/statements.
+
+```csharp
+public int Times3(int input) => input * 3;
+
+public int Interesting => 1729;
+```
+
+Expression-bodied-members cannot have blocks of multiple statements, but those with a functional background should be warned that anything that a traditional member can do can be achieved by one of these members. The "expression" can be an assignment operation creating side effects, or a method invocation meaning that anything is possible.
+
+#### Ternary Operators
+
+[Ternary operators][ternary-operators] allow if-conditions to be defined in expressions rather than statement blocks. This echoes functional programming approaches and can often make code more expressive and less error-prone.
```

Comment:
Maybe mention that a ternary always has an if and else clause?

Reply:
Done